nipype/interfaces/fsl/preprocess.py

```diff
@@ -315,6 +315,15 @@ def _format_arg(self, name, spec, value):
             formated = "-S %d %s" % (len(value), formated)
         return formated
 
+    def _parse_inputs(self, skip=None):
+        ''' ensures out_basename (argstring -o) always exists, otherwise fast
+        puts the outputs in the same folder as the inputs '''
```

Comment:
at the interface level, this is what it is supposed to do. within nipype we respect whatever the underlying interface does. 

however, at the workflow level, the files are copied/linked into the working directory, and fast should output the data in the correct place.